@xen-orchestra/web-core/lib/components/ui/table-actions/UiTableActions.vue

```diff
@@ -0,0 +1,40 @@
+<!-- v2 -->
+<template>
+  <div class="ui-table-actions">
+    <div class="actions">
+      <UiActionsTitle>
+        <slot />
+      </UiActionsTitle>
+      <slot name="actions" />
+    </div>
+    <div v-if="slots.groupedBy">
+      <slot name="groupedBy" />
+    </div>
```

Comment:
Since the component is named "Table actions", I think it would be more logical that it would take the "actions" in the default slot.

I propose to make the action slots default and change the current default to "title".

Also, I think we could add a title prop because in most cases we'll only want a simple text.

And this would add flexibility if for any reason we want to handle the `UiActionsTitle` component ourselves.

```suggestion
      <slot name="title">
        <UiActionsTitle>{{ title }}</UiActionsTitle>
      </slot>
      <slot />
    </div>
    <div v-if="slots.groupedBy">
      <slot name="groupedBy" />
    </div>
```